### DISCUSSION TITLE
Alexa Media Player perfectly connected but not saying a word

### DISCUSSION THREAD

**Discussion by varnone** [alandtse/alexa_media_player#1665] [Q&A]
Hello

I have just installed this fantastic component and the plugin is fully configured with my Alexa accoun so thumbs up for the developer. 

The strange thing is that I can see  that all devices are in the right status (Standby). If I device is playing something I can see that the device is in the right status. From the mini player I can even change the volume of the speaker. But when I try to make the device say something it does not work. I am based in Italy. I have double checked everthing and I have no clue on what to do. Any hints from this community ?

I have attached a link where you can see that my device is playing Rai 1 Radio [https://drive.google.com/file/d/1jdVkIi2TGg946lrWIMA6Oar0iOzGa_LU/view?usp=sharing](url)

Thanks in advance

Vincenzo

**Comment by varnone** [comment]
Hello

I have made some progress in this. It seams that I am able to send a message which is spoken only to an echo device that is not part of any group.

Sending a message to a device group or to a member of group does not work.

Any hint ?

Regards

Vincenzo

**Reply by alandtse** [reply]
https://github.com/custom-components/alexa_media_player/wiki/Configuration%3A-Notification-Component
